src/ml_board/frontend/newDashboard/src/webworkers/DedicatedWorkerClass.tsx

```diff
@@ -0,0 +1,42 @@
+type DedicatedWorkerClassInterface = {
+    worker: any // I don't know what type should come here. Ask someone regarding this...
+    userCallbacks: {
+        onMessageCtxNFunc: Function | null
+    }
+}
+
+class DedicatedWorkerClass implements DedicatedWorkerClassInterface{
+    
+    worker: any // I don't know what type should come here. Ask someone regarding this...
+    userCallbacks: {
+        onMessageCtxNFunc: Function | null
+    }
+
+    constructor(onMessageCtxNFunc = null) {
```

Comment:
do we really need a default parameter here? It should be always specified, no?

Reply:
Yes, it should always be specified. So made changes accordingly